test/comparisons.jl

```diff
@@ -87,8 +87,6 @@ end
             @test_throws MethodError setdiff(later, earlier)
             @test_throws MethodError symdiff(earlier, later)
 
-            # Using a vector of intervals as sets
-            @test union([earlier, later]) == [earlier, later]
```

Comment:
These tests should stay as they serve to showcase why you need `IntervalSet` when dealing with sets of `Interval`s. The comment should be updated to say:
```
# Test using a vector of intervals as a set only for comparison purposes
```
Unfortunately, I'm unable to suggest these changes.

Reply:
So you want to keep the test just to test the `Base` behaviour... and test the negation?

Reply:
Correct. For the same reason we test [vectors of intervals elsewhere](https://github.com/invenia/Intervals.jl/pull/214#discussion_r1206221922). Unless we can stop end users from using vectors of intervals seeing weird behaviour it's good to document/record the behaviour of vectors of intervals with the `Base` set functions.